exporter/datadogexporter/internal/translator/metrics_translator.go

```diff
@@ -306,7 +310,7 @@ func (t *Translator) mapHistogramMetrics(
 		case HistogramModeCounters:
 			t.getLegacyBuckets(ctx, consumer, name, p, delta, tags, host)
 		case HistogramModeDistributions:
-			t.getSketchBuckets(ctx, consumer, name, ts, p, delta, tags, host)
+			t.getSketchBuckets(ctx, consumer, name, startTs, ts, p, delta, tags, host)
```

Comment:
Here, `p` contains the `Timestamp` and `StartTimestamp`, so we shouldn't need to pass them to `getSketchBuckets`. We should do the same thing we're doing in `getLegacyBuckets` instead.

Reply:
Fixed, PTAL